Enums/HttpMethods.php

```diff
@@ -40,11 +41,12 @@ class HttpMethods {
      * @throws InvalidSqlOperationException
      *
      * @SuppressWarnings("PHPMD.StaticAccess")
+     * @SuppressWarnings("PHPMD.BooleanArgumentFlag")
      */
-    public static function ofSqlOperation($operation) {
+    public static function ofSqlOperation($operation, $patchInsert = false) {
```

Comment:
Thanks. I had the same dilemma when phpmd pointed it out. I couldn't see an alternative and it seemed perfectly within scope here.